codec-ntg/README.md

```diff
@@ -0,0 +1,13 @@
+# NTG codec
+
+This codec can be used for decoding and encoding messages in NTG (Native) protocol.
+
+## Configuration
+
+### Codec factory
+
+To use the NTG codec you will need to specify the following codec factory:
+**com.exactpro.sf.externalapi.codec.impl.ExternalNtgCodecFactory**
+
+### Implementation parameters
+This codec doesn't have any specific configuration. All decoding/encoding relies on the specified dictionary.
```

Comment:
This codec doesn't have any specific configuration. All **the** decoding/encoding relies on the specified dictionary.